lib/_emerge/DepPrioritySatisfiedRange.py

```diff
@@ -89,7 +89,10 @@ def _ignore_satisfied_buildtime_slot_op(cls, priority):
     def _ignore_runtime(cls, priority):
         if priority.__class__ is not DepPriority:
             return False
-        return bool(priority.satisfied or priority.optional or not priority.buildtime)
+        return (
+            bool(priority.satisfied or priority.optional or not priority.buildtime)
+            and not priority.runtime_slot_op
```

Comment:
We should allow it to be ignored here if priority.satisfied is True.

In DepPriorityNormalRange._ignore_runtime, we should also return False for priority.runtime_slot_op, and there it doesn't matter if it's satisfied or not.

Reply:
Looks good!